Libraries/Modal/Modal.js

```diff
@@ -77,6 +79,19 @@ class Modal extends React.Component {
   }
 }
 
+Modal.propTypes = {
+  animationType: PropTypes.oneOf(['none', 'slide', 'fade']),
```

Comment:
Wouldn't it be cool to export these constants as statics on Modal, so we can `Modal.AnimationType.Slide` and `Modal.AnimationType.Fade` etc ?